classes/book.rb

```diff
@@ -0,0 +1,14 @@
+# Define class Book
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_book_rental(book)
+    @rentals.push(book)
+  end
```

Comment:
- [ ] Here also we have to create a rental, so kindly add code to create rental here.
```ruby
 def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```